src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -1319,6 +1319,7 @@ private fun AnnotationSession.duplicatesByNamespace(
             + importedNames)
             .filter { it !is RsExternCrateItem } // extern crates can have aliases.
             .filter { it.nameOrImportedName() != null }
+            .filter { it.nameOrImportedName() != "_" }
```

Comment:
Maybe unite this and previous filter (`it.nameOrImportedName() != null`)

Reply:
Agree with @dima74